VagrantProvisionVars.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-export CHROMEDRIVER_VERSION=2.45
+export CHROMEDRIVER_VERSION=73.0.3683.68
```

Comment:
That's a big version jump.  Have we just neglected keeping it up to date?  Or did they jump versions?

Reply:
They changed to match the Chrome version instead of using their own custom version.   It went [from `2.x` to `70+` recently](https://chromedriver.storage.googleapis.com/index.html).